platforms/maven/opencv/pom.xml

```diff
@@ -175,14 +176,14 @@
                         <configuration>
                             <rules>
                                 <requireEnvironmentVariable>
-                                    <level>WARN</level>
+                                    <level>ERROR</level>
                                     <variableName>ANT_HOME</variableName>
                                     <message>$ANT_HOME is not set. Build may fail.</message>
```

Comment:
There is no requirement for ANT_HOME in Ubuntu 16.04 with system Apache ant package.
The same note about JAVA_HOME (it is not required with system java jdk package)